src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheck.java

```diff
@@ -72,6 +78,12 @@
      */
     private boolean ignoreOverridden;
 
+    /** The visibility scope where methods are checked. */
+    private Scope scope = Scope.ANONINNER;
```

Comment:
@agcuda , fyi https://github.com/checkstyle/checkstyle/issues/3511
sorry for mess in API. Please try to avoid ANONINNER as much as possible to simplify our future refactoring during API cleanup.